src/app/components/round-data-dialog/round-data-dialog.component.html

```diff
@@ -0,0 +1,11 @@
+<h2 mat-dialog-title>Round Data</h2>
+<div mat-dialog-content fxLayout="row" fxLayoutAlign="space-between">
+  Flowers visited:
+  <div *ngIf="this.flowersVisited$ | async as flowers; else zero">
+    {{flowers}}
+  </div>
+  <ng-template #zero>0</ng-template>
```

Comment:
```suggestion
  <div>
    {{(this.flowersVisited$ | async) ?? 0}}
  </div>
```

I think this would work.